http_urls.py

```diff
@@ -10,4 +11,5 @@
 
 urlpatterns = [
     path('', include(router.urls)),
+    path('application/<int:id>/secret_data/',ApplicationHiddenDetailView.as_view(),name="hidden-detail"),
```

Comment:
- Add one space to the right of each comma.
- ~"hidden-detail"~ 'hidden-detail'
- Break lines if the length exceeds more than 79 characters. (79 is allowed).